build.gradle.kts

```diff
@@ -25,27 +25,14 @@ repositories {
 kotlin {
   jvm("server") {
     compilations.all {
-      kotlinOptions.jvmTarget = "1.8"
+      kotlinOptions.jvmTarget = "11"
```

Comment:
I guess we still support jvm 8 - since there are no relevant benefits right now by using 11 we should stick at 8 ;)